docs/transfer/1-2-4-verify-authorization.puml

```diff
@@ -0,0 +1,64 @@
+@startuml
+
+title Transfer: 1.2.4 Verify Authorization
+
+
+box "Mojaloop"
+    participant Switch as S
+    participant "Auth-Service" as AUTHS
+end box
+box "DFSP A"
+    participant "DFSP A\n(Payer)" as D2
+end box
+
+
+rnote left of D2 #Light
+**""POST /thirdpartyRequests/verifications""**
+""FSPIOP-Source: dfspa""
+""FSPIOP-Destination: central-auth""
+{
+  "verificationRequestId": "44444444-0000-0000-0000-000000000000",
+  "challenge": ""<base64 encoded binary - the encoded challenge>"",
+  "value": {
+    "authenticationInfo": {
+      "authentication": "U2F",
+      "authenticationValue": {
+        "pinValue": "<base64 encoded binary - the signed challenge>",
+        "counter": "1"
+      }
+    }
+    "responseType": "ENTERED"
+  },
+  "consentId": "123"
+}
+end note
+D2 -> S: ""POST /thirdpartyRequests/verifications""
+S --> D2: ""202 Accepted""
+S -> AUTHS: ""POST /thirdpartyRequests/verifications""
+AUTHS --> S: ""202 Accepted""
+
+AUTHS -> AUTHS: Lookup this consent based on consentId
+AUTHS -> AUTHS: Ensure the accountAddress matches what is in Consent
```

Comment:
Ah I see. I think we can remove this point - there is no need to check the `accountAddress`, since the DFSP provides the `ConsentId`, which it finds based on the `payer` field in the `POST /thirdpartyRequests/transactions`

Reply:
TODO: Lewis to make a note about storing state of consentId etc. earlier on in sequences